### ISSUE TITLE
Generate cURL command for a `Case` instance

### ISSUE THREAD

**Issue by Stranger6667** [schemathesis/schemathesis#689]
As mentioned in #472

```python

@schema.parametrize()
def test_api(case):
    command = case.as_curl_command()
    ...
```

It should generate a string that could be copy-pasted and run in a terminal. The implementation itself might be similar to `Case.get_code_to_reproduce`, and the result should be compatible with the latest cURL behavior.

https://github.com/IBM/service-validator/pull/12 could be used as inspiration

**Comment by ZhukovGreen** [comment]
Hi @Stranger6667 I will try to help here

**Comment by ZhukovGreen** [comment]
Idea is to create a more complex app inside `test/apps` with all REST endpoints, then run generated curls and validate the responses. curl command produce similarly to https://github.com/IBM/service-validator/pull/12/files#diff-da75dd317c9eb18e4e459a0e5a67f18f31d887734af3471dddc07c94933cb737R198-R215

**Comment by Stranger6667** [comment]
> Idea is to create a more complex app inside test/apps with all REST endpoints, then run generated curls and validate the responses. curl command produced by https://github.com/ofw/curlify

It would be a great integration test!

**Comment by ZhukovGreen** [comment]
@Stranger6667 ok, then, I will do it in the next MR. I will fix your comments in #806 and let's merge it, so we can get some feedback. And I will imporve the testing separately. Thx for your feedback!
